app/stacks/appeals-service/variables.tf

```diff
@@ -15,6 +15,16 @@ variable "app_service_plan_id" {
   type        = string
 }
 
+variable "common_network_name" {
+  description = "The common infrastructure network name"
+  type        = string
+}
+
+variable "common_resource_group_name" {
```

Comment:
Could we call this `common_vnet_resource_group_name`? In case we end up with multiple resource groups in future.